docs/changelog.rst

```diff
@@ -1,6 +1,14 @@
 Change Log
 ==========
 
+1.0.2
+-----
+2015-07-07
+
+- Compatibility with Docker 1.7.0 by removing reliance on ENTRYPOINT.
+- Fix an issue with datapusher where the site_url would not be set 
+  correctly for CKAN to respond.
```

Comment:
this particular change doesn't seem to be on the branch. Let me find it and cherry-pick it as well